### PR TITLE
Modals and form errors

### PR DESCRIPTION
**Pull request by ericbieller**

### CODE REVIEW

File: src/less/buttons.css.less [hunk 1]
```diff
@@ -10,7 +10,7 @@ input[type="submit"],
 input[type="reset"],
 input[type="button"] {
   display: inline-block;
-  height: 42px;
+  height: 50px;
```
Comment (tommoor) [inline src/less/buttons.css.less:13]: check out the form when you click record call, this probably screwed the style there

File: src/less/forms.css.less [hunk 2]
```diff
@@ -49,7 +52,7 @@ input[type="password"]:focus,
 textarea:focus,
 select:focus {
   -webkit-app-region: no-drag;
-  border: 1px solid #4A90E2;
+  border: px solid @blue;
```
Comment (tommoor) [inline src/less/forms.css.less]: oddity here, still should be 1?

File: src/less/buttons.css.less (hunk 1 above)
Comment (ericbieller) [inline src/less/buttons.css.less:13]: Seems okay

image

**Comment by tommoor** [comment]
Cooooooo
- I feel like we should separate the channel name from the "join" text somehow maybe the channel name is heading and then some subtext on how they are joining the call? It's a little confusing just concatenated
- The error messages should animate in, fade in and slide down a little?
- Is it worth any other words on this page, feels very bare for someone that hasn't used the app before. A note on how we use their email perhaps?
- Personally preferred the "(optional)" on the right how you had it in the mocks

**Comment by tommoor** [comment]
Sexy AF, I'm sorry for playing with it :wink:

**Pull request merged**